buildpipeline/pipeline.json

```diff
@@ -335,6 +348,19 @@
             "Type": "build/product/",
             "ConfigurationGroup": "Debug"
           }
+        }, 
+        {
+          "Name": "DotNet-CoreFx-Trusted-Linux-Native",
+          "Parameters": {
+            "DockerTag": "ubuntu1404_prereqs_v3",
+            "portableLinux": "-portableLinux"
```

Comment:
That's correct. I can make the switch - I'm guessing I should do the same for CoreCLR?